test/test_batch_consistency.py

```diff
@@ -48,15 +48,12 @@ def test_griffinlim(self):
         )
 
     def test_detect_pitch_frequency(self):
-        filenames = [
-            'steam-train-whistle-daniel_simon.wav',  # 2ch 44100Hz
-            # Files from https://www.mediacollege.com/audio/tone/download/
-            '100Hz_44100Hz_16bit_05sec.wav',  # 1ch
-            '440Hz_44100Hz_16bit_05sec.wav',  # 1ch
+        waveform_and_sample_rates = [
+            torchaudio.load(common_utils.get_asset_path('steam-train-whistle-daniel_simon.wav')),  # 2ch 44100Hz
```

Comment:
Can you remove `steam-train-whistle-daniel_simon.wav` case?
I think we can cover pretty much the same thing with synthetic data.
We need to test the combinations of `sample_rate = 8000, 16000, 44100`, `n_channels = 1, 2`, and some frequency (100 and 440 should do). 

You can combine them using `parameterized.expand` and `itertools.product`. See https://github.com/pytorch/audio/blob/master/test/sox_io_backend/test_info.py#L24-L29